### PR TITLE
Classification levelling

### PR DESCRIPTION
**Pull request by spacecoffin**
Expands classification metrics and makes classification commands more modular.

### CODE REVIEW

File: cmd/files.go [hunk 1]
```diff
@@ -13,6 +14,41 @@ type Updater interface {
 	Update(y, yhat string)
 }
 
+type JsonMetrics struct {
+	ID        string
+	TP        int64
+	FP        int64
+	TN        int64
+	FN        int64
```
Comment (volker48) [inline cmd/files.go]: I'm not sure how you feel about this, but you could embed the whole ConfusionMatrix struct inside the JsonMetrics struct and all those fields would get put inside here.

Comment (spacecoffin) [reply cmd/files.go]: Comment addressed in latest commit.

File: cmd/files.go [hunk 2]
```diff
@@ -13,6 +15,37 @@ type Updater interface {
 	Update(y, yhat string)
 }
 
+type JsonMetrics struct {
+	*metrics.ConfusionMatrix
```
Comment (volker48) [inline cmd/files.go:19]: 👍

**Review by volker48 — APPROVED** [review]
Looks good. Go ahead and merge!

**Pull request merged**